.dockerignore

```diff
@@ -0,0 +1,5 @@
+# OpenShift carry: Copy .git to the container, build/build-all-amd.sh does `git rev-parse --short`
```

Comment:
nit: commit description could list all important changes like this one; since we're now using one OpenShift specific carry patch it might not be obvious what was done (apart from adding OpenShift files which is obvious from commit "header"). I believe this would be a full list:

-  removal of large files under build/oss dir (due to dist-git scripting)
- .gitignore modification
- build/build-all-amd.sh fixes
- .github removal